tests/python_package_test/test_dask.py

```diff
@@ -1253,5 +1257,9 @@ def test_sklearn_integration(estimator, check, client):
 # this test is separate because it takes a not-yet-constructed estimator
 @pytest.mark.parametrize("estimator", list(_tested_estimators()))
 def test_parameters_default_constructible(estimator):
-    name, Estimator = estimator.__class__.__name__, estimator.__class__
+    name = estimator.__class__.__name__
+    if sk_version > parse_version("0.23"):
+        Estimator = estimator
+    else:
+        Estimator = estimator.__class__
     sklearn_checks.check_parameters_default_constructible(name, Estimator)
```

Comment:
I did try just passing an instance always, so we could avoid parsing the scikit-learn version, but it looks like `scikit-learn` 0.22.x only supported passing a class in this method 😆 

Here's what I got changing the code to the following on `scikit-learn==0.22.0`

```python
@pytest.mark.parametrize("estimator", list(_tested_estimators()))
def test_parameters_default_constructible(estimator):
    name = estimator.__class__.__name__
    sklearn_checks.check_parameters_default_constructible(name, estimator)
```

> E           TypeError: 'DaskLGBMRegressor' object is not callable